examples/PhysicsLabFirmware/PhysicsLabFirmware.ino

```diff
@@ -40,6 +40,9 @@ BLECharacteristic              accelerationCharacteristic (SCIENCE_KIT_UUID("500
 BLECharacteristic              gyroscopeCharacteristic    (SCIENCE_KIT_UUID("5002"), BLENotify, 3 * sizeof(float));
 BLECharacteristic              magneticFieldCharacteristic(SCIENCE_KIT_UUID("5003"), BLENotify, 3 * sizeof(float));
 
+BLEService                     batteryservice             ("0x180F");
+BLEByteCharacteristic          batteryLevel               ("0x2A19", BLENotify);
```

Comment:
@facchinm the `0x` prefixed should be removed here, maybe this is causing issues?

Reply:
Ah, it may be the problem indeed, although I see the right characteristic on nRF connect...

Reply:
removing the 0x for me makes the UUID 16-bit on LightBlue, without it I see an 128-bit UUID.